inc/namespace.php

```diff
@@ -192,12 +192,21 @@ function unlink_cloudfront_distribution() {
 	delete_option( 'hm-cloudfront-origin-request-policy' );
 }
 
-function get_cloudfront_distribution_config() : array {
+/**
+ * Get the config for a Cloudfront distribution
+ *
+ * @param string|null $caller_reference The caller reference to pass to the distribution config.
+ * @return array The Cloudfront distribution config to pass to AWS.
+ */
+function get_cloudfront_distribution_config( $caller_reference = null ) : array {
 	$certificate = get_certificate();
 	$domains = array_unique( array_merge( [ $certificate['DomainName'] ], $certificate['SubjectAlternativeNames'] ) );
 	$cloudfront_function_arn = get_cloudfront_function_arn();
 	$origin_request_policy_id = get_cloudfront_origin_request_policy_id();
 
+	// If we haven't passed an explicit caller reference, then use the site url.
+	$caller_reference = $caller_reference ?? site_url();
+
 	$config = [
 		'CallerReference' => site_url(),
```

Comment:
I think we need this line to use `$caller_reference`.

Reply:
I cannot believe I missed that, I need to go and bury my head in the sand somewhere :sweat_smile: